quinn-proto/src/connection/mod.rs

```diff
@@ -542,15 +652,16 @@ where
                 }
             }
 
+            // Don't increment space_idx.
+            // We stay in the current space and check if there is more data to send.
+        }
+
+        // Finish the last packet. This one might also need to be padded
```

Comment:
It doesn't look like we're making a special effort to pad here, and calling out the usual machinery at one specific point seems a little odd.